CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ WYMeditor.
 
 *release-date* TBD
 
+* [#727](https://github.com/wymeditor/wymeditor/issues/727)
+  Added an "Upgrading to v1" documentation page
```

Comment:
The all important changelog entry. I've neglected the `### Doumentation` heading because this is the only item so far.